lib/seattleflu/id3c/cli/command/etl/redcap_det_scan.py

```diff
@@ -660,14 +686,13 @@ def collection_date(record: dict) -> Optional[str]:
     """
     Determine sample/specimen collection date from the given REDCap *record*.
     """
-    # The "PCDEQC" filled out by the unboxing team in the lab.  This instrument
-    # records the date written on the specimen tube if it does not match our
-    # other fields.  It is only used for mail-in samples.
-    qc_complete = is_complete('post_collection_data_entry_qc', record)
+    # The back_end_mail_scans is filled out by the logistics team for shipping.
+    # It is only used for mail-in samples.
+    back_end_complete = is_complete('back_end_mail_scans', record)
```

Comment:
I had to change this because the in-person projects now **do** have the PCDEQC instrument.

Reply:
Did they end up adding that instrument in for the older greek sweep project as well? (In case we had to reprocess)

Reply:
Yes, they did. It was added so that we have the "able_to_test" field in the project.

Reply:
Great! Thanks for making these updates 😄